tck/methods/account.go

```diff
@@ -0,0 +1,279 @@
+		}
+
+		_, err = transaction.FreezeWith(a.sdkService.Client)
+		if err != nil {
+			return nil, jrpc2.Errorf(-32603, err.Error())
+		}
+		transaction.Sign(key)
+	}
+
+	txResponse, err := transaction.Execute(a.sdkService.Client)
+	if err != nil {
+		fmt.Println(err)
+		return nil, jrpc2.Errorf(-32603, "Internal error").WithData(ErrorStatus{Status: err.Error()})
+	}
+	receipt, err := txResponse.GetReceipt(a.sdkService.Client)
+	var accId string
+	if receipt.Status == hedera.StatusSuccess {
+		accId = receipt.AccountID.String()
+	}
+
+	return &AccountResponse{AccountId: accId, Status: receipt.Status.String()}, nil
+}
+
+// CreateAccountFromAlias Create an account from aliasId by transferring  some HBAR amount from given account (opperatorId)
+func (a *AccountService) CreateAccountFromAlias(_ context.Context, fromAliasParams AccountFromAliasParams) (*AccountResponse, error) {
+	operator, err := hedera.AccountIDFromString(fromAliasParams.OperatorId)
+	alias, err := hedera.AccountIDFromString(fromAliasParams.AliasAccountId)
+
+	if err != nil {
+		return nil, jrpc2.Errorf(-32603, "Internal error", ErrorStatus{Status: err.Error()})
+	}
+
+	resp, err := hedera.NewTransferTransaction().SetGrpcDeadline(&threeSec).
+		AddHbarTransfer(operator, hedera.NewHbar(float64(fromAliasParams.InitialBalance)).Negated()).
+		AddHbarTransfer(alias, hedera.NewHbar(float64(fromAliasParams.InitialBalance))).
+		Execute(a.sdkService.Client)
+
+	if err != nil {
+		fmt.Println(err)
+		return nil, jrpc2.Errorf(-32603, "Internal error").WithData(ErrorStatus{Status: err.Error()})
+	}
+	receipt, err := resp.GetReceipt(a.sdkService.Client)
+	var accId string
+	if receipt.Status == hedera.StatusSuccess {
+		accId = receipt.AccountID.String()
+	}
+
+	return &AccountResponse{AccountId: accId, Status: receipt.Status.String()}, nil
+}
+
+// GetAccountInfo Get info for a given accountId and return a custom object containing aggregated information about the account
+func (a *AccountService) GetAccountInfo(_ context.Context, accountId string) (*AccountInfoResponse, error) {
+	account, _ := hedera.AccountIDFromString(accountId)
+
+	resp, err := hedera.NewAccountInfoQuery().SetAccountID(account).SetGrpcDeadline(&threeSec).Execute(a.sdkService.Client)
+	if err != nil {
+		return nil, jrpc2.Errorf(-32603, "Internal error", ErrorStatus{Status: err.Error()})
+	}
+
+	return &AccountInfoResponse{AccountID: resp.AccountID.String(),
+		Balance:                       resp.Balance.String(),
+		Key:                           resp.Key.String(),
+		AccountMemo:                   resp.AccountMemo,
+		MaxAutomaticTokenAssociations: resp.MaxAutomaticTokenAssociations,
+		AutoRenewPeriod:               resp.AutoRenewPeriod}, nil
+}
+
+type UpdateAccountParams struct {
+	AccountId string `json:"accountId"`
+	// Params for updateAccountKey
+	NewPublicKey  string `json:"newPublicKey"`
+	OldPrivateKey string `json:"oldPrivateKey"`
+	NewPrivateKey string `json:"newPrivateKey"`
+	// Params for updateAccountMemo
```

Comment:
No need to specify this. We could have endpoints that would need another variation. Comments regarding the purpose of the parameters here are irrelevant.